src/pyscript/_generator.py

```diff
@@ -140,26 +156,52 @@ def create_project(
 
     save_config_file(manifest_file, context)
 
-    index_file = app_dir / "index.html"
-    if project_type == "app":
-        template = "basic.html"
-    elif project_type == "plugin":
-        template = "plugin.html"
-    else:
-        raise ValueError(
-            f"Unknown project type: {project_type}. Valid values are: 'app' and 'plugin'"
-        )
+    if not wrap:
+        output = app_dir / "index.html"
+        if project_type == "app":
+            template = "basic.html"
+        elif project_type == "plugin":
+            template = "plugin.html"
+        else:
+            raise ValueError(
+                f"Unknown project type: {project_type}. Valid values are: 'app' and 'plugin'"
```

Comment:
Just a comment for a follow up perhaps but I think it would be good to put this in a consta and get this logic from somewhere that way we just need to add a new type and both the error message and template logic will stay the same